### PR TITLE
MCS - 1926 - Delete temporary files when browser session is ended

### PR DESCRIPTION
**Pull request by NDCACI**

### CODE REVIEW

File: webenabled/mcsweb.py [hunk 1]
```diff
@@ -217,6 +218,12 @@ def exit_unity():
     )
     resp.delete_cookie('uniq_id')
 
+    # delete static/mcsinterface/ folders
+    app.logger.info(f'{"Deleting static/mcsinterface/ folders"}')
+    os.system("find ./static/mcsinterface -name 'cmd_*' | xargs rm -r")
+    os.system("find ./static/mcsinterface -name 'output_*' | xargs rm -r")
+    os.system("find ./static/mcsinterface -name 'img_*' | xargs rm -r")
```
Comment (ThomasSchellenbergNextCentury) [inline webenabled/mcsweb.py:225]: For me, I don't see any `img_*` files in my `static/mcsinterface/` folder (is it an OS-specific thing?), but having it here doesn't hurt.

File: machine_common_sense/scripts/runner_script.py [hunk 2]
```diff
@@ -68,7 +68,7 @@ def __init__(
             )
             if args.save_videos:
                 subprocess.call([
-                    'ffmpeg', '-y', '-r', '20', '-i',
+                    'ffmpeg', '-y', '-r', '20', '-i', '-q:v', '1',
```
Comment (ThomasSchellenbergNextCentury) [inline machine_common_sense/scripts/runner_script.py:71]: Out of curiosity, what is this change for?

Comment (NDCACI) [reply machine_common_sense/scripts/runner_script.py:71]: Its something Phi changed while helping me with my import issue.

Comment (phile-caci) [reply machine_common_sense/scripts/runner_script.py:71]: Yikes... can you revert that, it was something I was testing.

**Review by rartiss55 — APPROVED** [review]

**Pull request merged**